trpcultivate_phenoshare/src/Plugin/TripalImporter/TripalCultivatePhenoShareImporter.php

```diff
@@ -125,11 +125,11 @@ class TripalCultivatePhenoShareImporter extends ChadoImporterBase implements Con
   protected $service_ConfigFactory;
 
   /**
-   * The Validator Plugin Manager.
+   * The TripalCultivate Base validator plugin manager.
    *
-   * @var \Drupal\trpcultivate_phenotypes\TripalCultivateValidator\TripalCultivatePhenotypesValidatorManager
+   * @var \Drupal\trpcultivate\TripalCultivateValidator\TripalCultivateValidatorManager
    */
-  protected TripalCultivatePhenotypesValidatorManager $service_validatorPluginManager;
+  protected TripalCultivateValidatorManager $service_BaseValidatorPluginManager;
```

Comment:
I'm not sure about this name including the word "Base". I think it could be accidently mixed up with `TripalCultivateValidatorBase` which inherits from `PluginBase`, and that holds a different meaning than when we are referring to the `TripalCultivate` module 😅 I personally would like to leave it as its previous name, `service_validatorPluginManager`, and we should assume that if we don't use the words 'Phenotypes', 'Genotypes', 'Germplasm', etc... then the service comes from the base module.

Reply:
> I personally would like to leave it as its previous name, service_validatorPluginManager, and we should assume that if we don't use the words 'Phenotypes', 'Genotypes', 'Germplasm', etc... then the service comes from the base module.

I agree with Carolyn on this :-) Both in this case and in moving forward as a general rule of thumb.